composer.json

```diff
@@ -18,7 +18,7 @@
     },
     "require": {
         "php": ">=5.4",
-        "symfony/framework-bundle": "~2.3",
+        "symfony/framework-bundle": "~2.7.5",
```

Comment:
Hum, it means that all Symfony versions prior to 2.7.5 would not be supported by this bundle. That's a problem, we do want to support them.